CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorRequestHandler.swift

```diff
@@ -114,7 +114,7 @@ public class ReplicatorRequestHandler {
 
         case "replicator_resetCheckpoint":
             let replication_obj: Replicator = args.get(name: "replicator")!
-            return replication_obj.resetCheckpoint()
+            return replication_obj.start(reset: true)
```

Comment:
this is not just a resetCheckpoint, this new API is _start_ with resetCheckpoint. So this will start a new replicator with reset checkpoint.